api/middleware_test.go

```diff
@@ -5,10 +5,14 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	
+	log "github.com/sirupsen/logrus"
```

Comment:
looks like this library is unused in the tests so its causing the test to fail

Reply:
(it was probably used in the benchmarks of the other codebase)

Reply:
Yep, fixed.